### DISCUSSION TITLE
Europe OSM import with Import_style=admin exceeds 380 GB, but in nominatim admin style import for entire planet is 220 GB.

### DISCUSSION THREAD

**Discussion by vigneshwarap** [mediagis/nominatim-docker#564] [Q&A]
Hi 

I am trying to import Europe OSM (around 28.6 GB) into my docker nominatim setup.
In official nominatim documentation it is mentioned that entire planet with admin style import is only 220 GB.[https://nominatim.org/release-docs/latest/admin/Import/]

But in mediagis/nominatim, europe import with import style as admin alone is 380GB. Other details given below.

Docker image version = 4.4
Docker container size after europe import  = 380GB (tested using "docker ps --size" command)
Started container using command = "docker run -it -e PBF_PATH=/nominatim/data/europe-latest.osm.pbf -e IMPORT_STYLE=admin -e REVERSE_ONLY=true -p 8080:8080 -v /root/vicky/OSM:/nominatim/data --name nominatim  mediagis/nominatim:4.4
"
OSM file downloaded from = https://download.geofabrik.de/

![Nominatim](https://github.com/mediagis/nominatim-docker/assets/39579257/ecb345a2-3a7e-468e-96c3-ed6f3e44096e)

Working with both docker and nominatim for just few days.
Am doing something wrong? Thanks in advance for your time.

**Comment by mtmail** [comment]
Have a look at https://github.com/mediagis/nominatim-docker/tree/master/4.4#flatnode-files

**Reply by vigneshwarap** [reply]
Hi @mtmail, Thanks for you response. Will try this configuration.

**Reply by vigneshwarap** [reply]
Hi @mtmail , Thanks for your help. flatnode-files seems to have reduced the size to one third.

These are my finds
![image](https://github.com/mediagis/nominatim-docker/assets/39579257/d5ad7af6-8c74-4df5-a385-9212dd4b9f3c)

Still this container sizes are huge.
Based on my analysis, Admin style import still imports address to village or suburb levels of accuracy. Is it possible to restrict import to city level to reduce container sizes.

My requirements are only for reverse geocoding to city level accuracy.

**Reply by mtmail** [reply]
https://github.com/mediagis/nominatim-docker/tree/master/4.4#import-style has link to the style format. You can create a new custom style for yourself based on the admin style.
